e2e-polybft/framework/test-server.go

```diff
@@ -226,7 +226,7 @@ func (t *TestServer) Stake(polybftConfig polybft.PolyBFTConfig, amount *big.Int)
 		"--" + polybftsecrets.AccountDirFlag, t.config.DataDir,
 		"--amount", amount.String(),
 		"--supernet-id", strconv.FormatInt(polybftConfig.SupernetID, 10),
-		"--native-root-token", polybftConfig.Bridge.RootNativeERC20Addr.String(),
+		"--stake-token", polybftConfig.Bridge.StakeTokenAddr.String(),
```

Comment:
Also here, get it from genesis?

Reply:
As per comment: https://github.com/0xPolygon/polygon-edge/pull/1520#discussion_r1200858670